Integration File,Start Date Time,Status,Description,End Date Time,Duration in Seconds,Rows Updated,Rows Created
eloqua_accounts.py,2021-11-09 12:08:58.759748,Pass,Successful Run,2021-11-09 12:09:04.937538,6.17779,11,17
eloqua_bounceback.py,2021-11-09 12:08:58,Fail,"(psycopg2.errors.DuplicateTable) relation ""t_eloqua_bounceback"" already exists

[SQL: CREATE TABLE customer360.t_eloqua_bounceback AS (SELECT * FROM customer360.s_eloqua_bounceback WHERE 1=2);]
(Background on this error at: http://sqlalche.me/e/f405)",2021-11-09 12:08:58,0,0,0
eloqua_contacts.py,2021-11-09 12:09:06.573517,Pass,Successful Run,2021-11-09 12:09:17.256200,10.682683,880,379
eloqua_emailclickthrough.py,2021-11-09 12:09:17.270208,Pass,Successful Run,2021-11-09 12:09:24.537999,7.267791,0,507
eloqua_emailopen.py,2021-11-09 12:09:24.548999,Pass,Successful Run,2021-11-09 12:09:49.599280,25.050281,0,13169
eloqua_emailsend.py,2021-11-09 12:09:49.614281,Pass,Successful Run,2021-11-09 12:10:08.033732,18.419451,0,10837
eloqua_formsubmit.py,2021-11-09 12:10:08.048734,Pass,Successful Run,2021-11-09 12:10:22.950371,14.901637,0,319
eloqua_pageview.py,2021-11-09 12:10:22.964373,Pass,Successful Run,2021-11-09 12:10:36.467081,13.502708,0,3482
eloqua_subscribe.py,2021-11-09 12:10:36.479084,Pass,Successful Run,2021-11-09 12:10:42.929910,6.450826,0,297
eloqua_unsubscribe.py,2021-11-09 12:10:42.941919,Pass,Successful Run,2021-11-09 12:10:49.129775,6.187856,0,297
eloqua_webvisit.py,2021-11-09 12:10:49.142776,Pass,Successful Run,2021-11-09 12:10:59.794556,10.65178,0,1333
salesforce_account.py,2021-11-09 12:11:00.200523,Pass,Successful Run,2021-11-09 12:11:07.181360,6.980837,115,55
salesforce_campaign.py,2021-11-09 12:11:07.192363,Pass,Successful Run,2021-11-09 12:11:13.247204,6.054841,259,23
salesforce_campaignmember.py,2021-11-09 12:11:13.257226,Pass,Successful Run,2021-11-09 12:11:37.873640,24.616414,2712,7257
salesforce_case.py,2021-11-09 12:11:37.875643,Pass,Successful Run,2021-11-09 12:11:56.261233,18.38559,7416,1471
salesforce_contact.py,2021-11-09 12:11:56.272239,Pass,Successful Run,2021-11-09 12:12:20.642685,24.370446,9303,407
salesforce_event.py,2021-11-09 12:12:20.656694,Pass,Successful Run,2021-11-09 12:12:32.630409,11.973715,179,307
salesforce_lead.py,2021-11-09 12:12:32.640410,Pass,Successful Run,2021-11-09 12:12:40.379218,7.738808,1826,783
salesforce_opportunity.py,2021-11-09 12:12:40.390226,Pass,Successful Run,2021-11-09 12:12:48.730041,8.339815,517,37
salesforce_recordtype.py,2021-11-09 12:12:48.740047,Pass,Successful Run,2021-11-09 12:12:52.213940,3.473893,116,0
salesforce_task.py,2021-11-09 12:12:52.223941,Pass,Successful Run,2021-11-09 12:13:09.390549,17.166608,5613,7457
salesforce_user.py,2021-11-09 12:13:09.402557,Pass,Successful Run,2021-11-09 12:15:29.203783,139.801226,128035,113
